fecgov/fec-cms
# Feature/4718 pac snapshot

## Summary

- Resolves #4718 

Building the PAC snapshot

### Required reviewers

UX, front-end (x2?), back-end to check the Python (I don't love this proposal)

## Impacted areas of the application

Adds the snapshot to the committees-single template, adds a couple new variables to get_committee in views.py

## Screenshots

(Include a screenshot of the new/updated features in context (“in the wild”). If it is an interface change, include both before and after screenshots)

## Related PRs

None

## How to test

- pull branch
- `npm run build`
- `./manage.py runserver`
- Check a ton of committee pages, sure to check different types (There have been several discussions and spreadsheets for good examples)

## Known issues
- I don't love this approach of determining the text to display for the glossary links
- There are still some glossary links that need guidance
